lib/std/build/run.zig

```diff
@@ -191,6 +191,13 @@ pub const RunStep = struct {
         child.stdout_behavior = stdIoActionToBehavior(self.stdout_action);
         child.stderr_behavior = stdIoActionToBehavior(self.stderr_action);
 
+        if (self.builder.verbose) {
```

Comment:
```suggestion
        if (self.builder.verbose and argv.len > 0) {
```

Reply:
I dont think this is needed. https://github.com/ziglang/zig/blob/fbda9991f48a6cd3a008890634ba4482f946a6f4/lib/std/build.zig#L329